qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,146 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario Outline: 1.1.2_01 User can successfully send CV
+	Given user inputted "<Name>", "<Email>", "<Checkbox>", "<Message>", attached <CV> in "Attach CV"
+	When user clicks Send button
+	Then message box "Thanks for being awesome! We have received your message and will come back to you as soon as we process it." should appear
+	And form should be empty
+
+	Examples:
+		| Name       | Email                 | Checkbox | Message               | CV         |
+		|Ivan Ivanov | ivan.ivanov@gmail.com | True     |                       | cv.pdf 5Mb |
+		|Ivan Ivanov | ivan.ivanov@gmail.com | True     | Hi, hire me please    | cv.pdf 5Mb |
+
+Scenario Outline: 1.1.2_02 User can't send form with empty required fields
+	Given user inputted "<Name>", "<Email>", "<Attach CV>", "<Checkbox>" 
+	When user clicks Send button
+	Then form shouldn't be sent, empty field should be highlighted in red
```

Comment:
It should be divided into 2 lines
	Then form shouldn't be sent, 
        And empty field should be highlighted in red